spec/seal_spec.rb

```diff
@@ -44,12 +61,28 @@
     context "when in quotes mode" do
       let(:teams) { [lions] }
 
-      it "barks at all teams" do
+      it "posts quotes if a quotes day" do
+        Timecop.freeze(Time.local(2023, 04, 24)) # Monday
         expect(Message).to receive(:new).with("go lions!").and_return(message)
         expect(SlackPoster).to receive(:new).with(lions.channel, anything)
 
         subject.bark(mode: "quotes")
       end
+
+      it "don't bark if not a quotes day" do
+        Timecop.freeze(Time.local(2023, 04, 25)) # Tuesday
+        expect(Message).not_to receive(:new).with("go dogs!")
+        expect(SlackPoster).not_to receive(:new).with(dogs.channel, anything)
+
+        subject.bark(mode: "quotes")
+      end
+
+      it "don't bark if unexpected values in quotes days" do
```

Comment:
I think for this test to work, you'll need to change line 45 to `let(:teams) { [lions, cats] }`